corehq/motech/repeaters/migrations/0006_state_only_rm_repeater_id.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.20 on 2023-09-17 15:22
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('repeaters', '0005_datasourcerepeater'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
```

Comment:
I understand that it would only affect django orm and not the actual DB, for my understanding can you please explain why it was done in this way?

Reply:
This allows us to make a change in the ORM without actually deleting the `repeater_id` column from the database (that will be done later). The column will be [populated by a trigger](https://github.com/dimagi/commcare-hq/blob/d133e1e8c8a8426021a9bafed7b14cea7eecae1c/corehq/motech/repeaters/migrations/0003_id_fields.py#L41-L51) in the mean time, so we could revert to using it if we discovered that we needed it before it is deleted.

The other operation (adding the UUID `id` column default) generates no SQL, so does not hurt to put it in here too. I could have left that one out of this block, but it didn't hurt to group it with the only other operation that affects the `Repeater` table.